src/index.js

```diff
@@ -0,0 +1,142 @@
+const state = {
+  tempValue: null,
+  increaseTempControl: null,
+  decreaseTempControl: null,
+  gardenLandscape: null,
+  tempValueEl: null,
+  cityLocation: {lat:  39.2776156 , lon: -74.574600}
```

Comment:
`cityLocation` is initialized here as an object with `lat` and `lon` keys. However, when you load the controls via `loadControls()` in the `setup()` function, it will immediately be overwritten and assigned to the result of `document.getElementById('cityNameInput')`. From what I can tell, you're typically referencing `cityLocation` as an object with `lat` and `lon` keys, so I'd suggest not overwriting in `loadControls`. Or perhaps I'm not seeing something?